inst/schema/NestedFilterOption.schema.json

```diff
@@ -8,7 +8,7 @@
     "options": {
       "type": "array",
       "items": {
-        "$ref": "#"
+        "type": "object"
```

Comment:
Annoyingly the generateTypeScript task in hint falls over with circular references in the json schema, whatever option you choose for supposedly dealing with them.